docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,117 @@
+---
+title: The newest additions to MUI X v6
+description: Support for time zones, a new filtering UI, charts! and more
+date: 2023-07-30T00:00:00.000Z
+authors: ['josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/card.png" alt="" style="margin-bottom: 16px;" width="2400" height="559" />
+
+Since the first v6 stable release, we have continuously rolled out new major features.
+Now that we reached [MUI X v6.10.0](https://github.com/mui/mui-x/releases/tag/v6.10.0), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Date Pickers](#date-pickers)
+  - [Support for Time Zones](#support-for-time-zones)
+  - [Digital Clock](#digital-clock)
```

Comment:
TimePicker, DateTimePicker speak a lot more to me than Digital Clock, the latter feels like an implementation detail, the former, what benefits my users.